docs/src/tricks.md

```diff
@@ -78,3 +78,26 @@ itself:
 imports = [ flake.inputs.stylix.nixosModules.stylix ];
 disabledModules = [ "${flake.inputs.stylix}/modules/<some-module>/nixos.nix" ];
 ```
+
+## Extending CSS options
+
+When trying to extend an attrset options the order does not matter because a
+delectation can only exists once. This is not the case for an option with the
+type of `lines` (most commonly `style` options in home-manager). For these options
+the order does matter and nix cannot guaranty that there aren't conflicting
+delectation. In order to get around this you can make sure nix puts your css at
+the end and thus prioritizes it with `lib.mkAfter`:
```

Comment:
Fixed typos, changed `home-manager` → `Home Manager`, `nix` → `Nix`, added commas:

```suggestion
When trying to extend an attrset option, the order does not matter because a
declaration can only exist once. This is not the case for an option with the
type of `lines` (most commonly `style` options in Home Manager). For these options,
the order does matter and Nix cannot guarantee that there aren't conflicting
definitions. In order to get around this, you can make sure Nix puts your CSS at
the end - and thus prioritizes it - by using `lib.mkAfter`:
```